CONTRIBUTING.rst

```diff
@@ -7,373 +7,372 @@ Contributing
-in order to make some of the new features available for testing on other projects before they
-are included in an actual full-blown release.
+To assess the quality of a transformer, we run quality tests that apply the Transformer
+on all the real world datasets that contain the Transformer input data type. The quality tests
+look at how well the original correlations are preserved by using transformed data to train
+regression models that predict other columns in the data. We compare the transformer's quality
+results to that of other transformers of the same data type.
 
-In order to perform such an action, you can execute::
+.. _Adding a Dataset:
 
-    make release-candidate
+Adding a Dataset
+""""""""""""""""
 
-This will perform the following actions:
+If the transformer you are creating adds a new data type, then a dataset with that type may need to
+be added for the quality tests. This only needs to be done if the transformer being added is 
+expected to preserve or expose relationships in the data. This can be done using the following
+steps:
 
-1. Build and upload the current version to PyPi as a pre-release, with the format ``X.Y.Z.devN``
+1. Find a dataset containing the data type your transformer uses as an input.
 
-2. Bump the current version to the next release candidate, ``X.Y.Z.dev(N+1)``
+2. Test your transformer against this dataset by loading it into a ``DataFrame`` and using the
+   ``get_transformer_regression_scores`` in the ``test_quality`` package::
 
-After this is done, the new pre-release can be installed by including the ``dev`` section in the
-dependency specification, either in ``setup.py``::
+    from tests.quality.test_quality import get_transformer_regression_scores
+    get_transformer_regression_scores(data, data_type, dataset_name, [transformer])
 
-    install_requires = [
-        ...
-        'rdt>=X.Y.Z.dev',
-        ...
-    ]
+3. If the scores are higher than the ``TEST_THRESHOLD`` in the ``test_quality`` package, contact
+   one of the `RDT core contributors`_ on GitHub and ask them to add the dataset. Once this is
+   done, the quality tests should pass.
 
-or in command line::
+Validating Quality
+""""""""""""""""""
 
-    pip install 'rdt>=X.Y.Z.dev'
+Validate the quality of your transformer using the
+``validate_transformer_quality`` function::
+
+    from tests.contributing import validate_transformer_quality
+    validate_transformer_quality('rdt.transformers.<YourTransformer>')
+
+Fix any quality issues that are reported.
+
+Finalize Your Transformer
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Re-run all the previous validations until they pass. For a final verification, run
+``validate_pull_request`` and fix any errors reported::
+
+    from tests.contributing import validate_pull_request
+    validate_pull_request('rdt.transformers.<YourTransformer>')
+
+Once you have done everything above, you can create a PR. Follow the steps below to create a PR.
```

Comment:
Can link to the `Pull Request Guidelines` here, since immediately below is the new transformer summary.